sequence_processing_pipeline/configuration.json

```diff
@@ -38,6 +38,20 @@
       "job_pool_size": 30,
       "job_max_array_length": 1000
     },
+    "nu-qc": {
+      "nodes": 1,
+      "nprocs": 8,
+      "queue": "qiita",
+      "wallclock_time_in_minutes": 240,
+      "minimap2_databases": "/panfs/cguccion/23_06_25_Pangenome_Assembley/downloaded_fastqs/fastq_files/pangenome_individual_mmi",
+      "modules_to_load": ["fastp_0.20.1", "samtools_1.12", " minimap2_2.18"],
```

Comment:
Not blocker but just out of curiosity, @wasade are these the same versions you are currently using? If not, are they modules or local installs?

Reply:
```
$ mamba list | egrep "minimap2|fastp|samtools"
fastp                     0.23.4               hadf994f_2    bioconda
minimap2                  2.26                 he4a0461_1    bioconda
samtools                  1.17                 hd87286a_1    bioconda
```